keras_cv/models/object_detection/retinanet/prediction_head.py

```diff
@@ -12,12 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from tensorflow import keras
-from tensorflow.keras import layers
+from keras_cv.backend import keras
 
 
-@keras.utils.register_keras_serializable(package="keras_cv")
-class PredictionHead(layers.Layer):
+@keras.saving.register_keras_serializable(package="keras_cv")
```

Comment:
`saving` should be preferred since it's the up to date namespace. But I've actually added support for `utils` for backwards compat too.